examples/03_second_level_models/plot_second_level_association_test.py

```diff
@@ -68,11 +68,55 @@
 _, threshold = map_threshold(z_map, alpha=.05, height_control='fdr')
 
 ###########################################################################
-#Let us plot the second level contrast at the computed thresholds 
+# Let us plot the second level contrast at the computed thresholds
 from nilearn import plotting
 plotting.plot_stat_map(
     z_map, threshold=threshold, colorbar=True,
     title='Group-level association between motor activity \n'
     'and reading fluency (fdr<0.05')
 
 plotting.show()
+
+##########################################################################
+# Computing the (corrected) p-values with parametric test to compare with
+# non parametric test
+from nilearn.image import math_img
+from nilearn.input_data import NiftiMasker
+p_val = model.compute_contrast('fluency', output_type='p_value')
+masker = NiftiMasker(mask_strategy='background').fit(p_val)
+n_voxel = np.size(masker.transform(p_val))
+# Correcting the p-values for multiple testing and taking neg log
+neg_log_pval = math_img("-np.log10(np.minimum(1, img * {}))"
+                        .format(str(n_voxel)),
+                        img=p_val)
+
+###########################################################################
+# Let us plot the second level contrast at the computed thresholds
+cut_coords = [38, -17, -3]
+plotting.plot_stat_map(
+    neg_log_pval, colorbar=True, cut_coords=cut_coords)
+plotting.show()
+
+##############################################################################
+# Computing the (corrected) p-values with permutation test
+from nistats.second_level_model import non_parametric_inference
+neg_log_pvals_permuted_ols_unmasked = \
+    non_parametric_inference(contrast_map_filenames,
+                             design_matrix=design_matrix,
+                             second_level_contrast='fluency',
+                             model_intercept=True, n_perm=1000,
+                             two_sided_test=False, mask=None,
+                             smoothing_fwhm=5.0, n_jobs=1)
+
+###########################################################################
+# Let us plot the second level contrast
+threshold_pval = 0
+plotting.plot_stat_map(
```

Comment:
Shouldn't we threshold at 1 ? Explaining that this is corresponds to p<.1 corrected.

Reply:
I don't use any treshold, otherwise the map is empty (even taking `threshold=1`). Should I let the code this way or do you have another suggestion ?

Reply:
Finally I made the change you suggested, but let me know, if you change your mind.